package.yml

```diff
@@ -1,14 +1,15 @@
 name: actnow-scrape
 version: 0.0.0
 author: Lukas Stevens <mail@lukas-stevens.de>
+url: https://github.com/lukasstevens/actnow-scrape
```

Comment:
@lukasstevens Do you want to rename the repo to `actnow-scrape`? Otherwise I'll change this back to `scrape-mep-mails`